js/localization/messages/fr.json

```diff
@@ -0,0 +1,239 @@
+        "dxPivotGrid-rowFieldArea": "Déposer les champs de ligne ici",
+        "dxPivotGrid-filterFieldArea": "Déposer les champs de filtre ici",        
+
+        "dxScheduler-editorLabelTitle": "Titre",
+        "dxScheduler-editorLabelStartDate": "Date de début",
+        "dxScheduler-editorLabelEndDate": "Date de fin",
+        "dxScheduler-editorLabelDescription": "Description",
+        "dxScheduler-editorLabelRecurrence": "Récurrence",
+
+        "dxScheduler-openAppointment": "Définir un évenement",
+
+        "dxScheduler-recurrenceNever": "Jamais",
+        "dxScheduler-recurrenceDaily": "Quotidien",
+        "dxScheduler-recurrenceWeekly": "Hebdomadaire",
+        "dxScheduler-recurrenceMonthly": "Mensuel",
+        "dxScheduler-recurrenceYearly": "Annuel",
+
+        "dxScheduler-recurrenceEvery": "Chaque",
+        "dxScheduler-recurrenceEnd": "Jusqu'à",
+        "dxScheduler-recurrenceAfter": "Après",
+        "dxScheduler-recurrenceOn": "Le",
+
+        "dxScheduler-recurrenceRepeatDaily": "Jour(s)",
+        "dxScheduler-recurrenceRepeatWeekly": "Semaine(s)",
+        "dxScheduler-recurrenceRepeatMonthly": "Mois(s)",
+        "dxScheduler-recurrenceRepeatYearly": "Année(s)",
+
+        "dxScheduler-switcherDay": "Jour",
+        "dxScheduler-switcherWeek": "Semaine",
+        "dxScheduler-switcherWorkWeek": "Semaine de travail",
+        "dxScheduler-switcherMonth": "Mois",
+
+        "dxScheduler-switcherAgenda": "Agenda",
+
+        "dxScheduler-switcherTimelineDay": "Timeline Jour",
+        "dxScheduler-switcherTimelineWeek": "Timeline Semaine",
+        "dxScheduler-switcherTimelineWorkWeek": "Timeline Semaine de travail",
+        "dxScheduler-switcherTimelineMonth": "Timeline Mois",
+
+        "dxScheduler-recurrenceRepeatOnDate": "le",
+        "dxScheduler-recurrenceRepeatCount": "occurence(s)",
+        "dxScheduler-allDay": "Temps plein",
+
+        "dxScheduler-confirmRecurrenceEditMessage": "Voulez-vous éditer cet évenement ou la série entière ?",
+        "dxScheduler-confirmRecurrenceDeleteMessage": "Voulez-vous supprimer cet évenement ou la série entière ?",
+
+        "dxScheduler-confirmRecurrenceEditSeries": "Editer serie",
+        "dxScheduler-confirmRecurrenceDeleteSeries": "Supprimer serie",
+        "dxScheduler-confirmRecurrenceEditOccurrence": "Editer évenement",
+        "dxScheduler-confirmRecurrenceDeleteOccurrence": "Supprimer évenement",
+
+        "dxScheduler-noTimezoneTitle": "Pas de fuseau horaire",
+
+        "dxCalendar-todayButtonText": "Aujourd'hui",
+        "dxCalendar-ariaWidgetName": "Calendrier",
+
+        "dxColorView-ariaRed": "Rouge",
+        "dxColorView-ariaGreen": "Vert",
+        "dxColorView-ariaBlue": "Bleu",
+        "dxColorView-ariaAlpha": "Transparence",
+        "dxColorView-ariaHex": "Code couleur",
+
+        "vizExport-printingButtonText": "Imprimer",
+        "vizExport-titleMenuText": "Exporterg/Imprimer",
```

Comment:
Hi, perrich!
"Exporterg" - is it the right translation? It seems there is a typo...

Reply:
Sorry for the delay, I've lost the notification :(

You're right. It's a typing issue, correct translation is "Exporter/Imprimer".